src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -138,45 +140,47 @@ export const styles = StyleSheet.create({
     flex: 1
   },
   pencilIconTouchable: {
-    padding: 4,
-    marginRight: 28,
+    padding: dimen.normalizeDimen(4),
+    marginRight: dimen.normalizeDimen(28),
     alignContent: 'center',
     alignItems: 'center',
-    height: 28
+    height: dimen.normalizeDimen(28)
   },
   gap: {
-    height: 1,
-    backgroundColor: COLORS.lightgrey
+    height: 2,
+    backgroundColor: COLORS.gray200
   },
   actionGroup: {
-    marginTop: 22
+    marginTop: dimen.normalizeDimen(6)
   },
   buttonGroup: {
     flexDirection: 'row',
-    paddingHorizontal: 20,
-    paddingVertical: 13
+    paddingHorizontal: dimen.normalizeDimen(20),
+    paddingVertical: dimen.normalizeDimen(13)
   },
   imageAction: {
-    height: 20,
-    width: 20
+    height: dimen.normalizeDimen(20),
+    width: dimen.normalizeDimen(20)
   },
   imageActContainer: {
-    marginRight: 26
+    marginRight: dimen.normalizeDimen(26)
   },
   textAct: {
     color: COLORS.redalert,
-    fontSize: 14
+    fontSize: normalizeFontSize(14),
+    lineHeight: normalizeFontSize(20),
+    fontFamily: fonts.inter[500]
   },
   mr7: {
     marginRight: dimen.normalizeDimen(12)
   },
   imageUser: {
-    height: normalize(48),
-    width: normalize(48),
-    borderRadius: normalize(24)
+    height: dimen.normalizeDimen(48),
+    width: dimen.normalizeDimen(48),
+    borderRadius: dimen.normalizeDimen(24)
   },
   parentContact: {
-    height: normalize(72)
+    height: dimen.normalizeDimen(72)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The normalization function `dimen.normalizeDimen` is used instead of hardcoded values for styling. This change improves consistency in the codebase by using the same normalization function throughout.
```diff
- padding: 4,
- marginRight: 28,
- height: 28
+ padding: dimen.normalizeDimen(4),
+ marginRight: dimen.normalizeDimen(28),
+ height: dimen.normalizeDimen(28)

- height: 1,
- backgroundColor: COLORS.lightgrey
+ height: 2,
+ backgroundColor: COLORS.gray200

- marginTop: 22
+ marginTop: dimen.normalizeDimen(6)

- paddingHorizontal: 20,
- paddingVertical: 13
+ paddingHorizontal: dimen.normalizeDimen(20),
+ paddingVertical: dimen.normalizeDimen(13)

- marginRight: 26
+ marginRight: dimen.normalizeDimen(26)

- marginRight: dimen.normalizeDimen(12)
+ marginRight: dimen.normalizeDimen(12)

- height: normalize(48),
- width: normalize(48),
- borderRadius: normalize(24)
+ height: dimen.normalizeDimen(48),
+ width: dimen.normalizeDimen(48),
+ borderRadius: dimen.normalizeDimen(24)

- height: normalize(72)
+ height: dimen.normalizeDimen(72)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->